servo_controll.py

```diff
@@ -0,0 +1,37 @@
+import RPi.GPIO as GPIO
+import time
+
+number_plate_detected_entering = True
+number_plate_detected_leaving = True
+auto_detected_entering = True
+auto_detected_leaving = True
+
+servoPIN1 = 13
+servoPIN2 = 14
+pulse_frequency = 50
+GPIO.setmode(GPIO.BOARD) #we use pin numbers
+GPIO.setup(servoPIN1, GPIO.OUT)
+GPIO.setup(servoPIN2, GPIO.OUT)
+
+servo1 = GPIO.PWM(servoPIN1, pulse_frequency)
+servo2 = GPIO.PWM(servoPIN2, pulse_frequency)
+
+def setAngle(angle, servo):
+    duty = angle/18 + 2
+    servo.ChangeDutyCycle(duty)
+    time.sleep(1)
+
+if number_plate_detected_entering:
+    setAngle(90, servo1)
+    while auto_detected_entering:
+        time.sleep(1)

```

Comment:
This can be quite dangerous, because this locks the program in this infinite look when `auto_deteced_entering` is `False`. That way, not other processes can be started. But we do have to decide how we're going to handle multiple processes on the Pi.